tests/repository/test_publish.py

```diff
@@ -146,7 +152,9 @@ def test_publish_with_options(requests_mocker, client):
         ],
     )
 
-    options = PublishOptions(clean=True, force=True, origin_only=True)
+    options = PublishOptions(
+        clean=True, force=True, origin_only=True, rsync_extra_args=frozenlist(["-a"])
```

Comment:
Can you please not explicitly initialize it with a frozenlist?

We want it to work with just rsync_extra_args=["-a"] and for the conversion to a frozenlist to be a hidden implementation detail.  Initializing without `frozenlist` would be the typical case, so that's what the test should do.

Reply:
Ah, I missed this. Yeah, it should just be a list.